.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,20 @@
+name: Jest Tests
```

Comment:
You can delete this github/workflows directory to prevent CI from running for this exercise